cfpb/consumerfinance.gov
# Update to design-system 0.43.1, cypress@13.7.3, eslint-plugin-jsdoc@48.2.3 , snyk@1.1288.0

Design-system changes https://github.com/cfpb/design-system/compare/v0.43.0...v0.43.1

## Changes

- Update to design-system 0.43.1, cypress@13.7.3, eslint-plugin-jsdoc@48.2.3 , snyk@1.1288.0
- TCCP: Fix error of unused `event` argument.


## How to test this PR

1. `yarn install && yarn build` notifications should not have extra margin at mobile size.
